polaris-react/src/components/TextField/TextField.scss

```
@@ -127,6 +127,43 @@ $spinner-icon-size: 12px;
 }
 // stylelint-enable
 
+// stylelint-disable selector-max-specificity, selector-max-class, selector-max-combinators, selector-max-compound-selectors -- specificity bump for magic styles to override
+.magic {
+  > .Input {
+    color: var(--p-color-text-magic);
+  }
+
+  > .Backdrop {
+    // stylelint-disable-next-line -- --new-magic-generated-surface
+    background-color: rgba(248, 247, 255, 1);
```

```suggestion
    background-color: var(--p-color-bg-surface-magic);
```

We'll update the value of this token to match the designs